README.md

```diff
@@ -2,13 +2,13 @@
 
 # A Package for Exploratory Data Analysis (eda_analysis)
 
-![](https://github.com/sweber15/eda_analysis/workflows/build/badge.svg) 
+![build](https://github.com/UBC-MDS/edapython/workflows/build/badge.svg) 
 
-![codecov](https://codecov.io/gh/sweber15/eda_analysis/branch/master/graph/badge.svg)
+[![codecov](https://codecov.io/gh/UBC-MDS/edapython/branch/master/graph/badge.svg)](https://codecov.io/gh/UBC-MDS/edapython)
  
 ![Release](https://github.com/sweber15/eda_analysis/workflows/Release/badge.svg)
```

Comment:
Sorry that I might be wrong, but is this button also needed fixed?

Reply:
yes. The previous button didn't point to a valid button.

Reply:
Oh I see you're referring to the release. That hopefully will be fixed after this changes gets merged, which then runs `release.yml` successfully.

Reply:
actually, let me fix that.